account-management-api/src/main/java/uk/gov/di/accountmanagement/entity/AuthenticateResponse.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.di.accountmanagement.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AuthenticateResponse extends BaseAPIResponse {
+
+    @JsonProperty("redactedPhoneNumber")
+    private String redactedPhoneNumber;
+
+    public AuthenticateResponse(
+            @JsonProperty(required = true, value = "redactedPhoneNumber")
```

Comment:
i would just return a 200 without anything else as all the information already exists in the front end session